sdk_contrib/fetch/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aws-xray-sdk-fetch",
+  "version": "3.5.3",
+  "description": "AWS X-Ray plugin for node-fetch",
+  "author": "Amazon Web Services",
+  "contributors": [
+    "Jason Terando <jasonterando@yahoo.com>"
+  ],
+  "main": "lib/fetch_p.js",
+  "files": [
+    "lib"
+  ],
+  "types": "lib/fetch_p.d.ts",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "test": "mocha --recursive --exit ./test",
```

Comment:
Can you update this to match the other script tests
`"test": "mocha --recursive --exit ./test/ -R spec && tsd",`

Reply:
Done

Reply:
Done